payu/models/cice.py

```diff
@@ -201,9 +194,55 @@ def setup(self):
             setup_nml['runtype'] = 'continue'
             setup_nml['restart'] = True
 
+        else:
+            # Locate and link any restart files (if required)
+            if not setup_nml['ice_ic'] in ('none', 'default'):
+                self.link_restart(setup_nml['ice_ic'])
+
+            if setup_nml['restart']:
+                self.link_restart(setup_nml['pointer_file'])
+
+        self.calc_runtime(setup_nml)
+
+        # Write any changes to the work directory copy of the cice
+        # namelist
+        nml_path = os.path.join(self.work_path, self.ice_nml_fname)
+        self.ice_in.write(nml_path, force=True)
+
+    def calc_runtime(self, setup_nml):
```

Comment:
```suggestion
    def _calc_runtime(self, setup_nml):
```

preceding underscore to indicate its a "private" function (even though python doesn't do private)

Reply:
Good idea! I'll commit this outside of the github editor to add in the changes to the method calls.

Reply:
Fixed up in f5ab3b4